lib/ttnt/test_to_code_mapping.rb

```diff
@@ -15,22 +14,23 @@ def initialize(sha)
     end
 
     def append_from_coverage(test, coverage)
-      spectra =  normalize_path(select_project_files(spectra_from_coverage(coverage)))
-      save_spectra(test: test, spectra: spectra)
+      spectra = normalize_path(select_project_files(spectra_from_coverage(coverage)))
+      save_mapping(test: test, spectra: spectra)
     end
 
-    def read_spectra(test:)
-      if File.exists?(spectra_file(test: test))
-        JSON.parse(File.read(spectra_file(test: test)))
+    def read_mapping
+      if File.exists?(mapping_file)
+        JSON.parse(File.read(mapping_file))
       else
         {}
       end
     end
 
+    # Get tests affected from change of file `file` at line number `lineno`
     def get_tests(file:, lineno:)
       tests = Set.new
-      all_tests.each do |test|
-        spectra = read_spectra(test: test)
+      mapping = read_mapping
```

Comment:
Oops, thank you :smile: 
I fixed it here: https://github.com/Genki-S/ttnt/commit/a10adf95730ed86b9dd1c39663e80a557566445f
(though it might be a little messy since that p/r should be improvement for #6 ... please forgive me :sweat_smile: )